src/Microsoft.TemplateSearch.Common/BlobTemplateInfo.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Utils;
+using Newtonsoft.Json.Linq;
+
+#nullable enable
+
+namespace Microsoft.TemplateSearch.Common
+{
+    internal class BlobTemplateInfo : ITemplateInfo, IShortNameList
```

Comment:
Do we need the blob data to be stored as a `ITemplateInfo`? I imagine we can get away with a simple type with some basic fields like name, description etc.

Reply:
I agree we don't need `ITemplateInfo` for search